libsspl_sec/tests/acceptance/helpers/base64.c

```diff
@@ -25,7 +25,7 @@
 #include <math.h>
 #include <openssl/bio.h>
 #include <openssl/evp.h>
-BIO_METHOD* BIO_f_base64(void);
+const BIO_METHOD* BIO_f_base64(void);
```

Comment:
Hi @cdeshmukh, I need to remove this line or comment it out, since
`BIO_METHOD* BIO_f_base64(void);` This method is declared as `non const` in `openssl-1.0.2` and as `const` in `openssl-1.1.1`. so if I keep this declaration its going to cause a build error in either el7.x or in el8.x. 
In el7.x it is not possible to upgrade to openssl-1.1.1 that's why.

Reply:
Reply on stak-overflow.
![image](https://user-images.githubusercontent.com/45289079/126941059-7f41792e-d592-49a4-91f5-9e81c258743e.png)

Reply:
Okay, so on systems coming pre-installed with openssl-1.1.1, only there 'BIO_METHOD* BIO_f_base64(void)' can declared const !?
Rocky Linux doesnt come with openssl-1.1.1 ? as its pretty new OS distribution already ?

Reply:
No no. EPEL 7 is for el7, in centos 7.x  we can not upgrade to openssl-1.1.1.
That function declaration is already there in openssl irrespective of the version. so we can remove that, 
my question was whether to keep it as commented for future references or simply remove it.
```BIO_METHOD* BIO_f_base64(void)' can declared const !?```
about declaration, we need to be const in rocky and non-const in centos, to be clear.

Reply:
Can remove, shouldn't be a issue as declaration is there in openssl/evp.h

Reply:
removed.